spooky-doodle/Trestlebridge-Farms
# Chickens and ducks

# Description

Users can purchase animals after creating a field appropriate for that animal type.  If a facility is full or no appropriate field exists, and error will be displayed and the user will be returned to the main menu.

Fixes #2 #5 #7 #9 #10 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1.  From a fresh start of the application: Create a facility (1), select either Grazing Field (1), Chicken House (4), or Duck house (5).  Then purchase an animal (2) appropriate for the field that was created.  The list of available facilities should be displayed with the current number of animals in the field.  Select the desired field, and the animal should be added.

2.  From a fresh start of the application: Purchase an animal (2).  Select any animal (1-7).  An error message should be displayed and the user will be returned to the main menu.



# Checklist:
